dlp/snippets/src/main/java/dlp/snippets/TriggersPatch.java

```diff
@@ -0,0 +1,94 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dlp.snippets;
+
+// [START dlp_update_trigger]
+
+import com.google.cloud.dlp.v2.DlpServiceClient;
+import com.google.privacy.dlp.v2.InfoType;
+import com.google.privacy.dlp.v2.InspectConfig;
+import com.google.privacy.dlp.v2.InspectJobConfig;
+import com.google.privacy.dlp.v2.JobTrigger;
+import com.google.privacy.dlp.v2.JobTriggerName;
+import com.google.privacy.dlp.v2.Likelihood;
+import com.google.privacy.dlp.v2.UpdateJobTriggerRequest;
+import com.google.protobuf.FieldMask;
+import java.io.IOException;
+
+public class TriggersPatch {
+
+  public static void main(String[] args) throws Exception {
+    // TODO(developer): Replace these variables before running the sample.
+
+    // The Google Cloud project id to use as a parent resource.
+    String projectId = "your-project-id";
+    // The name of the job trigger to be updated.
+    String jobTriggerName = "your-job-trigger-name";
+    patchTrigger(projectId, jobTriggerName);
+  }
+
+  public static void patchTrigger(String projectId, String jobTriggerName) throws IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    //
+    try (DlpServiceClient dlpServiceClient = DlpServiceClient.create()) {
+
+      // Specify the type of info the inspection will look for.
+      // See https://cloud.google.com/dlp/docs/infotypes-reference for complete list of info types
+      InfoType infoType = InfoType.newBuilder().setName("PERSON_NAME").build();
+
+      InspectConfig inspectConfig = InspectConfig.newBuilder()
+              .addInfoTypes(infoType)
+              .setMinLikelihood(Likelihood.LIKELY)
+              .build();
+
+      // Configure the inspection job we want the service to perform.
+      InspectJobConfig inspectJobConfig = InspectJobConfig.newBuilder()
+              .setInspectConfig(inspectConfig)
+              .build();
+
+      JobTrigger jobTrigger = JobTrigger.newBuilder()
+              .setInspectJob(inspectJobConfig)
+              .build();
+
+      FieldMask fieldMask = FieldMask.newBuilder()
+              .addPaths("inspect_job.inspect_config.info_types")
+              .addPaths("inspect_job.inspect_config.min_likelihood")
+              .build();
+
+      // Combine configurations into a request for the service.
+      UpdateJobTriggerRequest updateJobTriggerRequest = UpdateJobTriggerRequest.newBuilder()
+              .setName(JobTriggerName.of(projectId, jobTriggerName).toString())
+              .setJobTrigger(jobTrigger)
+              .setUpdateMask(fieldMask)
+              .build();
+
+      // Send the scan request and process the response
```

Comment:
Update this comment; we are not sending a scan request, just updating a job trigger.

Job Trigger updates do not immediately cause scans, and job trigger results are not returned in api but need to be fetched separately later.